segmentation_labeling_app/transforms/transformations.py

```diff
@@ -91,105 +91,107 @@ def _downsample_array(full_array: h5py.Dataset, input_fps: int,
-        # can't be centered left
-        if coordinate_pair[0] < (math.ceil(box_size[0]/2) - 1):
-            transformed_row = math.ceil(box_size[0] / 2) - 1
-
-        # can't be centered up
-        if coordinate_pair[1] < (math.ceil(box_size[1]/2) - 1):
-            transformed_col = math.ceil(box_size[1] / 2) - 1
-
-        # can't be centered right
-        if (coordinate_pair[0] + (math.ceil(box_size[0]/2))) > video_shape[0] - 1:
-            transformed_row = video_shape[0] - math.floor(box_size[0] / 2) - 1
-
-        # can't be centered down
-        if (coordinate_pair[1] + (math.ceil(box_size[1]/2))) > video_shape[1] - 1:
-            transformed_col = video_shape[1] - math.floor(box_size[1] / 2) - 1
-
-    return transformed_row, transformed_col
-
-
-def get_centered_coordinate_box_video(coordinate_pair: Tuple[int, int],
-                                      box_size: Tuple[int, int],
-                                      video_array: np.ndarray):
-    """
-    Function to get video subset centered around coordinate pair with
-    size specified by box size tuple. Function takes a subset of each frame
-    and stacks together to get final video.
-    Args:
-        coordinate_pair: the coordinate on which to center the subset (row, col)
-        box_size: the size of the subset box (row, col)
-        video_array: the video to take the subset from array should be of
-                     form (time, row, col)
-
-    Returns:
-
-    """
-    if not len(video_array[0].shape) == 2:
-        raise ValueError('Video does not have correct shape')
-    transformed_coordinates = get_transformed_center(coordinate_pair, box_size,
-                                                     video_array[0].shape)
-    left_column = transformed_coordinates[1] - math.floor(box_size[1] / 2)
-    right_column = transformed_coordinates[1] + math.ceil(box_size[1] / 2)
-    up_row = transformed_coordinates[0] - math.floor(box_size[0] / 2)
-    down_row = transformed_coordinates[0] + math.ceil(box_size[0] / 2)
-    transformed_video = np.zeros(shape=(video_array.shape[0], box_size[0], box_size[1]))
-    for i, frame in enumerate(video_array):
-        transformed_video[i] = frame[up_row:down_row, left_column:right_column]
-    return np.uint8(transformed_video)
-
-
-def generate_max_ave_proj_image(video: np.ndarray,
-                                projection_type: str = 'average'):
-    """
-    Returns a maximum projection or an average projection of a video in
-    numpy array format
-    Args:
-        video: The video to generate the projection with shape (time, row, col)
-        projection_type: maximum or average, what type of projection
-
-    Returns: A numpy array generated with the specified strategy (time, row, col)
-
-    """
-    if projection_type == 'average':
-        return np.mean(video)
-    elif projection_type == 'maximum':
-        return np.max(video, axis=0)
+# def get_transformed_center(coordinate_pair: Tuple[int, int],
```

Comment:
If this is no longer required, I think we should delete rather than comment.

Reply:
agreed, just waiting to see where chips fall

Reply:
If it's working for your runs I think we should delete it in this PR.

Reply:
removed comment blocks.